DocumentDataAPI/DocumentDataAPITests/Data/Repositories/NpgSearchRepositoryIntegrationTests.cs

```diff
@@ -77,4 +77,45 @@ public async Task Get_QueryNotFound_ReturnsEmptyList()
         // Assert
         result.Should().BeEmpty("because no documents contain the term \"videnskab\"");
     }
+
+    [Fact]
+    public async Task GetAll_WithSpecificLimitAndOffset_ReturnsConsistentResults()
+    {
+        //Arrange
+        List<string> query = new() { "dronningen", "døde", "og" };
+        const int limit = 3;
+        const int firstOffset = 0,
+            nextOffset = 3;
+
+        //Act
+        IEnumerable<SearchResponseModel> firstResult = await _repository.Get(query, new DocumentSearchParameters(), limit, firstOffset);
+        IEnumerable<SearchResponseModel> nextResult = await _repository.Get(query, new DocumentSearchParameters(), limit, nextOffset);
+
+        //Assert
+        firstResult.Should().SatisfyRespectively(
+            x => x.DocumentModel.Id.Should().Be(1L),
+            x => x.DocumentModel.Id.Should().Be(2L),
+            x => x.Relevance.Should().Be(0)
+        );
+        nextResult.Should().HaveCount(2)
+            .And.AllSatisfy(x => x.Relevance.Should().Be(0));
+    }
```

Comment:
Hvorfor er den her ikke to seperate tests?

Reply:
Testen skal sørge for at hvis du laver samme søgning, så er resultaterne altid consistent når du bruger limit og offset. Hvis der f.eks. ikke blev sorteret på primary key, så kunne vi risikere at `firstResult` og `nextResult` _ikke_ passer sammen som  forventet i forskellige kørsler af denne test.